pico_mrp/models/pico_workflow.py

```diff
@@ -134,6 +134,8 @@ def _sync_data(self, values):
         workflow = self.search([('pico_id', '=', workflow_pico_id)], limit=1)
         if not workflow:
             workflow = self.create(self._get_values_from_pico_data(workflow_data))
+        else:
+          workflow.write({'name': workflow_data.get('name', '')})
```

Comment:
The change makes sense to me. Looks like your indentation isn't quite right. I have my vim config setup to auto format python files with `black` which might help there (though with the default settings, `black` would change a lot of lines - mostly around moving single quotes to double quotes).